test/functional/fixtures/regression/gh-993/test.js

```diff
@@ -0,0 +1,5 @@
+describe('[Regression](GH-993)', function () {
+    it("Key events should have 'key' or 'keyIdentifier' properties when t.pressKey('enter') performed", function () {
```

Comment:
Key events should have the 'key' or 'keyIdentifier' property when t.pressKey('enter') is performed